meta-ostro-xt/recipes-opencl/ocl-icd/ocl-icd/configurable-doc-generation.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/configure.ac b/configure.ac
```

Comment:
Thanks

1. we have reported to meta-intel-realsense. 2. is ours but it looks correct. Any idea why it fails?